index.html

```diff
@@ -936,7 +935,7 @@ <h3>Relationship to Verifiable Credential Type Property</h3>
           </p>
           <p>
             The example below illustrates this for <code>EmailCredential</code>:
-            <pre class="example" title="Verifiable Credential with Schema Type">
+            <pre class="example vc-jwt" title="Verifiable Credential with Schema Type">
```

Comment:
`vc-jwt` -> `vc-jose-cose` ?

Reply:
the plugin won't understand this... If `vc-jose-cose` can make progress, I will rewrite the plugin to better handle the changes since it was originally written.